source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/edit/index.tsx

```diff
@@ -24,30 +24,33 @@ export interface IOwnProps {
 
 export type IProps =
   & IOwnProps
+  & WithDataProps
   & i18nProps;
 
 class Profile extends React.Component<IProps> {
 
-  state = {
-    imageData: null
-  };
-
-  onChangePicture = (imageData) => {
-    this.setState({imageData});
-  }
-
   updateProfile = async (formData: UserAttributes): Promise<void> => {
     const { t, user } = this.props;
     const id = idFor(user);
 
     try {
-      const { imageData } = this.state;
 
       await this.props.updateStore(tr => tr.replaceRecord({
         id,
         type: TYPE_NAME,
-        attributes: { ...formData, imageData }
-      }), defaultOptions());
+        attributes: { ...formData }
+      }), {
+        sources: {
+          remote: {
+            settings: {
+              headers: {
+                ...defaultHeaders(),
+                ['content-type']: 'application/vnd.api+json'
```

Comment:
is this needed?
what happens if you replace this entire sources object with defaultOptions()?